packages/hoprd/Dockerfile

```diff
@@ -1,74 +1,66 @@
-# Run HOPRd  within a single container.
+# Run HOPRd  within a single container using npm
 
-# -- BASE STAGE --------------------------------
-
-# Based on https://snyk.io/blog/10-best-practices-to-containerize-nodejs-web-applications-with-docker/
-# From the #1 recommendation “Use explicit and deterministic Docker base image tags”
-
-# Docker image builds are inconsistent. Just like we’re using lockfiles to get a deterministic
-# npm install behavior every time we install npm packages, we’d also like to get deterministic
-# docker image builds. If we build the image from node—which effectively means the node:latest
-# tag—then every build will pull a newly built Docker image of node.
-
-# Originally node:12.9.1-buster, obtained by
-# $ docker pull node:12.9.1-buster
-# $ docker images --digests
-# and added the sha to the image.
-FROM node:14.15-buster@sha256:74a322375cbf1bfca73e3d41f39d4a04e4869c7defdf83f9663166989d1b553c AS base
-WORKDIR /hoprd
+# use slim version of node on Debian buster for smaller image sizes
+FROM node:14-buster-slim@sha256:ccc25181fd9bc0170cfa2f873b11d005d41c06a1f4fa9e64aa89c579bcc6b03b as build
 
+# python is used by some nodejs dependencies as an installation requirement
 RUN apt-get update && \
     apt-get install -y \
-    libssl-dev \
-    ca-certificates \
-    fuse \
-    gcc \
-    cmake \
-    wget
+    python3 \
+    build-essential
 
-ENV YARN_VERSION 1.19.2
-RUN yarn policies set-version $YARN_VERSION
-COPY package.json .
-RUN yarn install --build-from-source
+WORKDIR /app
 
-# -- BUILD STAGE --------------------------------
+# enable to pass the version to Docker using either --build-arg or an
+# environment variable
+# if its not given, yarn will install the latest version of the package
+ARG HOPRD_VERSION
+ENV HOPRD_VERSION=${HOPRD_VERSION:-}
 
-FROM base as build
+# making sure some standard environment variables are set for production use
+ENV NODE_ENV production
+ENV NEXT_TELEMETRY_DISABLED 1
+ENV DEBUG 'hopr*'
```

Comment:
This doesn't seem to be doing anything, I have to manually add  `--env="DEBUG=hopr*"` when running the resulting image, otherwise I get not output.

Reply:
will fix